lib/bootloader_setup.pm

```diff
@@ -249,7 +250,24 @@ sub boot_local_disk {
         if (match_has_tag 'inst-slof') {
             diag 'specifying local disk for boot from slof';
             my $slof = 5;
-            $slof += 1 if get_var('VIRTIO_CONSOLE');
+            if (get_var('VIRTIO_CONSOLE') && is_ppc64le) {
+                type_string_very_slow "devalias";
+                send_key 'ret';
+                save_screenshot;
+                my $file = './serial0';
```

Comment:
Can you explain more on this file? As I know it is defined in qemu backend
` "-chardev ringbuf,id=serial0,logfile=serial0,logappend=on -serial chardev:serial0"`

My question is that, does this configuration have something to do with VIRTIO_CONSOLE?

2nd question, if we are in the same directory name as serial0 file, can we just set
`my $file = serial0`